starknet_devnet/blueprints/postman.py

```diff
@@ -55,3 +57,30 @@ async def flush():
 
     result_dict = await state.starknet_wrapper.postman_flush()
     return jsonify(result_dict)
+
+
+@postman.route("/l1_to_l2", methods=["POST"])
```

Comment:
I think the name of this endpoint (and function) is misleading. Why not call it `/message_to_l2` or `/send_message_to_l2`?

Reply:
fixed in https://github.com/Shard-Labs/starknet-devnet/pull/365/commits/28c4600564e0d6d863a79f59b5b178831c8fa70e